src/main/java/com/habicus/core/entities/Goal.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of the Habicus Core Platform (https://github.com/Habicus/Habicus-Core).
+ * Copyright (c) 2018 Habicus Core
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.habicus.core.entities;
+
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/** Goal entity that will reference */
+@Entity
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "goal")
+public class Goal {
+
+  @Id
+  @GeneratedValue
+  private Long id;
+
+  private String title;
```

Comment:
Why do these not have @XmlElement annotations?

Reply:
I'm assuming it's because it defaults to the name and accountable is different than the name?

Reply:
When adding the `@XmlAccessorType(XmlAccessType.FIELD)` annotation to the class, it will auto associate member variables to the name in the XML unless otherwise specified. Adding an explicit `name` annotation essentially says the XML and member variable aren't named the same.

Reply:
Ok that's what I was thinking. Thanks.

Reply:
Your assumption is correct.